tools/ktx/command_create.cpp

```diff
@@ -1329,6 +1350,16 @@ void CommandCreate::executeCreate() {
                 image->yflip();
             }
 
+            if (options.normalize) {
+                if (target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
```

Comment:
For future proofing

```
if (target.format().transfer() != KHR_DF_TRANSFER_UNSPECIFIED && target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
```

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/977/commits/61556ccf343abe68c01e8497f00c9ca2f786170c